sandbox/pyodide/build_packages.sh

```diff
@@ -16,11 +16,13 @@ echo ""
 echo "###############################################################"
 echo "## Prepare python packages"
 
+source env.sh
 cd _build/pyodide
-git checkout 0.23.4 || (git fetch && git checkout 0.23.4)
+git checkout $PYODIDE_VERSION || (git fetch && git checkout $PYODIDE_VERSION)
```

Comment:
Above the script starts with `set -e`.

What do you think of adding the `-u` option? I think it can be especially useful when you are using variables:
```
-u  Treat unset variables as an error when substituting.
```

(usually I work with `set -eEuo pipefail` which covers many cases of error)